### ISSUE TITLE
[Bug] : Percent value is converted to a different value

### ISSUE THREAD

**Issue by mischnic** [cssnano/cssnano#957]
**Describe the bug**
`70%` is minified into `1%`

**To Reproduce**

1. Go to https://cssnano.co/playground/
2. Insert `body {	opacity: 70%; }` on the left
3. "Run"
4. See incorrect result

**Input**
```css
body {opacity: 70%;}
```

**Current behaviour**
```css
body{opacity:1%}
```

**Expected behavior**
```css
body{opacity:70%}
```

**Desktop (please complete the following information):**
- cssnano: 4.1.10
- cssnano-preset-default: 4.0.7
- postcss-convert-values 4.0.1

**Additional context**
This seems to be fixed by using this config:
```js
module.exports = {
	preset: [
		"default",
		{
			convertValues: false,
		},
	],
};
```

**Comment by anikethsaha** [comment]
Thanks for the issue

This has been fixed in master. 
It will be released soon. 
Till then use `cssnano@nightly`

**Comment by ludofischer** [comment]
Fixed in 5.0.0-rc.2
